laravel-app/resources/views/profile/edit.blade.php

```diff
@@ -1,18 +1,35 @@
 @extends('layouts.app')
 
-@section('content')
-
-@include('partials._header')
-<div id="page-content">
-	<div class="p-3">
-		<div class="row justify-content-center">
-			<div class="col-4 p-5 bg-light">
-				@include('profile.form')
-			</div>
-		</div>
-	</div>
-</div>
-
-@include('partials._footer')
+@php
+    $authUser = auth()->user();
+    $imagePath = $authUser && $authUser->media->isNotEmpty() ? 'storage/' . $authUser->media->last()->file_path : 'assets/images/user-solid.svg';
+@endphp
 
```

Comment:
Transfer this to your controller.

Reply:
Already transferred it sir to controller